storio-sample-app/src/main/java/com/pushtorefresh/storio/sample/db/resolvers/PersonPutResolver.java

```diff
@@ -0,0 +1,43 @@
+package com.pushtorefresh.storio.sample.db.resolvers;
+
+import android.support.annotation.NonNull;
+
+import com.pushtorefresh.storio.sample.db.entities.Person;
+import com.pushtorefresh.storio.sample.db.tables.CarsTable;
+import com.pushtorefresh.storio.sample.db.tables.PersonsTable;
+import com.pushtorefresh.storio.sqlite.StorIOSQLite;
+import com.pushtorefresh.storio.sqlite.operations.put.PutResolver;
+import com.pushtorefresh.storio.sqlite.operations.put.PutResult;
+import com.pushtorefresh.storio.sqlite.operations.put.PutResults;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static java.util.Arrays.asList;
+
+public class PersonPutResolver extends PutResolver<Person> {
+
+    @NonNull
+    @Override
+    public PutResult performPut(@NonNull StorIOSQLite storIOSQLite, @NonNull Person person) {
+        // We can even reuse StorIO methods
+        final PutResults<Object> putResults = storIOSQLite
+                .put()
+
+                // TODO here I need help
+                .objects(asList(person, person.getCars()))
```

Comment:
You can not put `person` via `storIOSQLite.put()` because it will be recursive, so you need to use `storIOSQLite.internal()`, but you can use `storIOSQLite.put()` to put `cars`.